modules/monitoring-example-remote-write-authentication-settings.adoc

```diff
@@ -27,8 +27,8 @@ monitoring user-defined projects
 endif::openshift-dedicated,openshift-rosa[]
 in the `{namespace-name}` namespace.
 
-.Sample YAML for AWS Signature Version 4 authentication
-====
+== Sample YAML for AWS Signature Version 4 authentication
```

Comment:
I am not sure what benefit come from promoting these to a heading. [All headings in module files](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings) take an anchor ID.